webapp/src/main/webapp/news-list-view/components/views/NewsMosaicView.vue

```diff
@@ -91,6 +96,9 @@ export default {
     isMobile() {
       return this.$vuetify.breakpoint.name === 'xs' || this.$vuetify.breakpoint.name === 'sm';
     },
+    isSmallBreakpoint() {
+      return this.$vuetify.breakpoint.width < 651;
```

Comment:
we can not introduce a new viewport, we need to use existing ones . could you make needed changes ?